Gopkg.lock

```diff
@@ -1694,6 +1694,7 @@
     "github.com/docker/go-metrics",
     "github.com/ghodss/yaml",
     "github.com/gobuffalo/packr/v2",
+    "github.com/gobuffalo/packr/v2/file/resolver",
```

Comment:
We are still improving our makefile targets to help prevent this package from sneaking in here but in the meantime, it's going to cause the CI build to fail when it runs `make verify`. To avoid triggering this in the future, before running `dep ensure`, run `make clean-packr`. Sorry about that!

Once you you can reset this file, you should be fine, and I'll kick off the build for you again.